staff/static/staff/css/sidebar.css

```diff
@@ -5,4 +5,13 @@
 
 .sidebar-link-active {
     border-width: 4px !important;
+}
+
+.topbar a {
```

Comment:
Likely not necessary now but in future, wondering if we want to include SASS in this project to use CSS variables and selector nesting.